docs/components/FileInputView.less

```diff
@@ -0,0 +1,7 @@
+@import (reference) "~less/index";
+
+
+.TextInputView--inputContainer {
```

Comment:
Maybe make this selector include `.FileInput` somehow to avoid messing with the styles for `TextInputView` elsewhere? Not totally sure where this is being used, btw